routes.yaml

```diff
@@ -0,0 +1,7 @@
+app_name: acsp-manage-users-api
+group: api
+
+weight: 100
+# TODO: check this
+routes:
+  1: ^/acsp-associations/*
```

Comment:
we will have 3 routes.
/acsp-members/*
/internal/acsp-members/*
/acsp-manage-users-api/healthcheck

Reply:
Updated routes.